tests/test_layers.py

```diff
@@ -59,11 +69,12 @@ def run_test_wait_depth(self,layer):
     def run_test_resources(self,layer):
         # check resources
         rsc = layer.resource()
-        self.assertEqual(set(list(rsc.keys())), set(["BRAM","DSP","LUT","FF"]))
+        # self.assertEqual(set(list(rsc.keys())), set(["URAM", "BRAM","DSP","LUT","FF"]))
         self.assertTrue(rsc["LUT"] >= 0)
         self.assertTrue(rsc["FF"] >= 0)
         self.assertTrue(rsc["DSP"] >= 0)
         self.assertTrue(rsc["BRAM"] >= 0)
+        # self.assertTrue(rsc["URAM"] >= 0)
```

Comment:
We could maybe enable or disable this line based on the fpga that is loaded for a specific run/experiment and its support of URAM or not

Reply:
@AlexMontgomerie Is there any update here? If not lets resolve the conversation